### DISCUSSION TITLE
Error - Hotkeys and write text actions require you are in the input group

### DISCUSSION THREAD

**Discussion by mintgitb1ker** [streamdeck-linux-gui/streamdeck-linux-gui#231] [Q&A]
Hotkeys were working, now I get this - Missing Permissions and a link to this page - https://github.com/streamcontroller/osplugin?tab=readme-ov-file#hotkeys--write-text

But running the code on that page in terminal does not change the issue (KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess", GROUP="input", MODE="0660")
Am I entering that code wrong? Missing out something?
btw Streamcontroller is amazing at speeding things up.

**Comment by coolapso** [comment]
I guess you opened the discussion on the wrong Project? ;)

**Reply by mintgitb1ker** [reply]
yep - I am an idiot

**Comment by Core447** [comment]
I updated the instructions. If this still doesn't work, please open a new discussion on the [StreamController repo](https://github.com/StreamController/StreamController).
